src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -324,22 +383,21 @@ public static class PublishedContentExtensions
         /// <returns>The <see cref="object"/> representing the Umbraco value.</returns>
         /// <param name="instance">The instance to assign the value to.</param>
         private static object GetRawValue(
-           IPublishedContent content,
-            Type type,
+            IPublishedContent content,
             CultureInfo culture,
             PropertyInfo propertyInfo,
-            ref object instance)
+            object instance)
```

Comment:
Just noticed that `ref` has been removed here. @mattbrailsford Does this have an impact?